scapy/supersocket.py

```diff
@@ -185,6 +185,55 @@ def tshark(self, *args, **kargs):
         from scapy import sendrecv
         return sendrecv.tshark(opened_socket=self, *args, **kargs)
 
+    def _command(self, d, callbacks=None):
+        """Internal helper function for implementing the command() method.
+
+        :param d: a dict with all parameters passed to the constructor
+            of the class, where a key is the name of the parameter
+            in the constructor and the value is its value
+        :param callbacks: a dict, where a key is the name of a parameter
+            in the constructor of the class and the value is a callback
+            function that accepts the value of the parameter and returns
+            a string representation of it
+        """
+
+        nv = []
+        for n, v in six.iteritems(d):
+            if v is None:
+                continue
+            elif hasattr(v, "__iter__"):
+                try:
+                    c = len(v) == 0
+                except TypeError:
+                    c = False
+                if c:
+                    continue
+
+            # provide a callback if all other cases,
```

Comment:
If you don't use the callbacks, better remove them.

Reply:
Right